### ISSUE TITLE
TypeError: Cannot read property 'getDOMNode' of undefined

### ISSUE THREAD

**Issue by kateboyko** [react-materialize/react-materialize#375]
Get such error while adding Modal to the page.
the same error appears at your sample page https://react-materialize.github.io/#/modals

```
Uncaught TypeError: Cannot read property 'getDOMNode' of undefined
    at OverlayTrigger.getContainerDOMNode (Overlay.js:95)
    at OverlayTrigger._mountOverlayTarget (Overlay.js:65)
    at OverlayTrigger._renderOverlay (Overlay.js:71)
    at OverlayTrigger.componentDidMount (Overlay.js:59)
    at CallbackQueue.notifyAll (CallbackQueue.js:76)
    at ReactReconcileTransaction.close (ReactReconcileTransaction.js:80)
    at ReactReconcileTransaction.closeAll (Transaction.js:206)
    at ReactReconcileTransaction.perform (Transaction.js:153)
    at batchedMountComponentIntoNode (ReactMount.js:126)
    at ReactDefaultBatchingStrategyTransaction.perform (Transaction.js:140)
```

**Comment by Haitrancntt** [comment]
Hi guy, i have the same problem, still don't have any solution for this

**Comment by alextrastero** [comment]
Sorry guys, seems like we've been having plenty of issues with the Modals lately, https://github.com/react-materialize/react-materialize/pull/377

**Comment by alextrastero** [comment]
Will publish as soon as reviewed

**Comment by Haitrancntt** [comment]
ok, hope it completed soon. Thanks

**Comment by alextrastero** [comment]
@kateboyko @Haitrancntt this is now resolved in `1.0.14`

**Comment by Haitrancntt** [comment]
Nice, let me check this, thanks
